alert/SAPHanaSR-alert-fencing

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 #
 # SAPHanaSR-alert-fencing
-# Author:       Lars Pinne Fabian Herschel, June 2024
+# Author:       Lars Pinne, Fabian Herschel, June 2024
 # Support:      linux@sap.com
 # License:      GNU General Public License (GPL)
-# Copyright:    (c) 2024 SUSE LLC
-# Version:	2024-08-09-14:35
+# Copyright:    (c) 2024-2025 SUSE LLC
+# Version:	    2024-07-29-17:50
```

Comment:
the date changed into the past. May be you want 2025 instead of 2024?

Reply:
fixed